### PR TITLE
OS-6580 LX fails to handle futex_requeue correctly

### PR DESCRIPTION
**Pull request by jasonbking**

### CODE REVIEW

File: usr/src/uts/common/brand/lx/syscall/lx_futex.c [hunk 1]
```diff
@@ -1401,12 +1401,23 @@ lx_futex(uintptr_t addr, int op, int val, uintptr_t lx_timeout,
...
 	case FUTEX_CMP_REQUEUE:
-	case FUTEX_REQUEUE:
 		rval = futex_requeue(&memid, &memid2, val,
 		    val2, (void *)addr2, &val3);
 
 		break;
 
+	case FUTEX_REQUEUE:
+		/*
+		 * Per linux futex(2), FUTEX_REQUEUE is the same as
```
Comment (papertigers) [inline usr/src/uts/common/brand/lx/syscall/lx_futex.c:1411]: Capitalize Linux. (existing lx style preference)

**Review by danmcd — APPROVED** [review]
Seems sensible

**Comment by papertigers** [comment]
@jasonbking do you want to run the ltp test suite or would you like help doing that?

**Comment by jasonbking** [comment]
Do you have something already setup to run it? If so it'd probably be faster for you to run it -- I've not run it before (at least that I can recall), so I'd need to figure all of that out.

**Review by papertigers — APPROVED** [review]
The ltp run looked good as well.  And node is verified to exit correctly under musl based distributions now.

**Comment by danmcd** [comment]
Thanks for making sure the test results are in the ticket.

**Pull request merged**